contracts/registry/Registry.sol

```diff
@@ -117,6 +154,62 @@ contract Registry is AdminRole {
         return maxTrusts[_adr];
     }
 
+    /// @notice Return the pending balance of an address, or 0 if the address is not a contributor.
+    /// @param _adr (address) Address to check
+    /// @return pendingBalance (uint256) Pending balance of the address, or 0 if not a contributor.
+    function getPendingBalance(address _adr)
```

Comment:
Perhaps `getContributorInfo` should return pending balance also?

Reply:
Actually I thought about it, the doubted it may broke some agreement and used api!
I am up to change it if it's safe

Reply:
Should be reasonable safe since call signature doesn't change. I doubt function is called outside Commons Stack codebase. Only found the following uses, a utility script and a test case:

https://github.com/commons-stack/cstk-token/blob/94a73886246500ddf155080b8f778fecf44d5c1a/scripts/trustedAccounts.ts#L11

https://github.com/commons-stack/cstk-token/blob/94a73886246500ddf155080b8f778fecf44d5c1a/test/unit/Registry.spec.ts#L147